README.md

```diff
@@ -1,72 +1,62 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Javascript Group captsone
 
-> Description the project.
+> Javascript Group captsone is a project built with Joffrey and Vincent at Javascript Module completion. It has many functionalities where a user visit the homepage. the application loads data from two APIs and display them on interface. when a user click on heart icon it increments the number of likes on that specific item. when a user click on comments button It shows project descriptions and its comments. it gives also the functionality of adding new comments to the Item.
 
+##Video Link 
+[video Link](https://drive.google.com/file/d/1gykehauOaQ9MKBOsU18KaYj4WFxPWBKd/view?usp=sharing)
+[Live Demo](https://joffreynk.github.io/Module2-FInalMealCapstone/)
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- HTML
+- CSS
+- JAVASCRIPT
+- WEBPACK
+- API
 
-## Live Demo (if available)
-
-[Live Demo Link](https://livedemo.com)
 
 
 ## Getting Started
 
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
-
-
-To get a local copy up and running follow these simple example steps.
-
-### Prerequisites
+**clone the project and run **npm i** to install depencies.**
+**after installing dependancies run npm start**
```

Comment:
- [ ] Please include code to clone the repo.For ex:-
`git clone https://github.com/JoffreyNK/Module2-FInalMealCapstone.git`
`cd Module2-FInalMealCapstone`